lib/services/nodes-api-service.js

```diff
@@ -84,6 +84,8 @@ function nodeApiServiceFactory(
      * @return {Object}    node after removing relation
      */
     NodeApiService.prototype._removeRelation = function removeRelation(node, type, targets) {
```

Comment:
Since it is used in api/2.0/nodes, it might be better to remove the "_" prefix, which indicates a private function. Same as _addRelation